### ISSUE TITLE
Target field in MSI-installed Windows shortcut 

### ISSUE THREAD

**Issue by terencebarrett** [pybee/briefcase#53]
With Briefcase 0.1.9 I was able to create a windows installer (MSI) for the temperature conversion app example, but after installing from the MSI, I needed to change the shortcut's Target field from
"C:\Program Files (x86)\Freedom\python\pythonw.exe" app\start.py
to
"C:\Program Files (x86)\Freedom\python\pythonw.exe" "C:\Program Files (x86)\Freedom\app\start.py"
for the program to run. (Windows 7 and Windows Server 2008)

**Comment by freakboy3742** [comment]
@terencebarrett Thanks for the report! Out of interest - how did you start the app? The shortcut *should* have had a working directory of `c:\Program Files (x86)\Freedom`, so the extra prefix you added shouldn't have been required. Was the "start in" option empty after your installation?

**Comment by terencebarrett** [comment]
@freakboy3742 Upon install, for the two fields of interest, I get:
Target: "C:\Program Files (x86)\Freedom\python\pythonw.exe" app\start.py
Start in: "C:\Program Files (x86)\Freedom\app\"
Which doesn't launch.

Target: "C:\Program Files (x86)\Freedom\python\pythonw.exe" start.py
Start in: "C:\Program Files (x86)\Freedom\app"
Also doesn't launch.

What you are expecting it to be:
Target: "C:\Program Files (x86)\Freedom\python\pythonw.exe" app\start.py
Start in: "C:\Program Files (x86)\Freedom"
Does launch.

I enjoyed the BeeWare tutorial at PyCon. Since then I have continued to play with it (including successfully repeating the tutorial in a conda environment), have became an Enthusiast, and am working to get my team interested in the project.
